test/integration/sessions/sessions.spec.test.js

```diff
@@ -3,11 +3,11 @@
 const path = require('path');
 const { expect } = require('chai');
 const { TestRunnerContext, generateTopologyTests } = require('../../tools/spec-runner');
-const { runUnifiedTest } = require('../../tools/unified-spec-runner/runner');
+const { runUnifiedSuite } = require('../../tools/unified-spec-runner/runner');
 const { loadSpecTests } = require('../../spec');
 
-describe('Sessions', function () {
-  describe('legacy spec tests', function () {
+describe('Sessions spec tests', function () {
+  describe('Sessions legacy spec tests', function () {
```

Comment:
I think this goes back to what @nbbeeken said about having repeated blocks with the same names even if they are in different files, if it doesn't cause an issue for the runner, then yeah, we can remove it

Reply:
All our top level suites need to have unique names in order to generate the final xunit file. (I think this is actually just a limitation I introduced by using a Map in the reporter, so we could circle back on that) It uses `suite.fullTitle()` to identify each suite, so every nested suite will have its parent's name prefixed, so even if there's multiple `legacy spec tests` strings, as long as their parent's are different we're fine. 

In fact, (don't wanna bloat more sorry, but) https://github.com/mongodb/node-mongodb-native/blob/main/test/tools/reporter/mongodb_reporter.js#L178 we could change this warn to throw now, when I first wrote the reporter we had many dupes, but we've since cleared them up in follow up work